php-classes/PasswordAuthenticator.class.php

```diff
@@ -41,6 +43,16 @@ public function checkAuthentication()
             $this->_authenticatedPerson = $this->attemptAuthentication($requestData['username'], $requestData['password']);
 
             if ($this->_authenticatedPerson) {
+                
+                // set cookie for username
+                if (static::$rememberUsernameEnabled && isset($requestData['remember'])) {
```

Comment:
@jtwiest use `!empty(...)` instead of `isset()` so that if a blank value gets posted it won't be interpreted as being enabled